confd/confd.go

```diff
@@ -0,0 +1,17 @@
+package confd
+
+
+// RawData is a map of raw data, it can be used to unmarshal json data
+type RawData map[string]interface{}
+
+type Order map[string]int
+
+func Contains(s []interface{}, e interface{}) bool {
```

Comment:
**[checkstyle]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function Contains should have comment or be unexported